README.md

```diff
@@ -50,6 +54,24 @@ def main():
 if __name__ == "__main__":
     main()
 ```
+##### Configure the default answer
+Add to `slackbot_settings.py` a default_reply:
+```python
+default_reply = "Sorry but I didn't understood you" 
```

Comment:
It would be more flexible if `default_reply` can be either a string or a callable.

- If it's a string, we just send it back to slack, as you have implemented.
- If it's a callable, it's supposed to take a single argument - the `Message` object.